### ISSUE TITLE
Flit doesn't work for characters not in view

### ISSUE THREAD

**Issue by zim0369** [ggandor/flit.nvim#37]
Follow up to this discussion https://github.com/LazyVim/LazyVim/discussions/1100#discussioncomment-6428126

If I try to run `t` or `f` on any character that's not visible in the current window, then it wouldn't jump to that character.

**Comment by jyscao** [comment]
Yes, perhaps some users prefer to only be able to navigate to characters that are on-screen. But I think this should at least be made available as an option, especially for those of us that prefer to use `nowrap`.
